File: Assignment__13/Program3.c
//_______________________________________________________________________________________________________________________________
//
//  File name :     Program3.c
//  Description :   Accept number of rows and number of columns from user and display below pattern.
//
//                          Input :  iRows = 3       iCol = 5
//
//                          Output :   A  A  A  A  A
//                                     B  B  B  B  B
//                                     C  C  C  C  C
//                                     
//  Author :        Sarika Devidas Gadhe
//  Date :          20/05/2025
//
//_______________________________________________________________________________________________________________________________

#include<stdio.h>

void Pattern (int iRow, int iCol)
{
    int i = 0,j = 0;
    char ch = '\0';

    for(i = 1,ch = 'A';i <= iRow; i++,ch++)
    {
        for(j = 1; j <= iCol; j++)
        {
            printf("%c\t",ch);
        }
        printf("\n");

    }
}
int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("Enter number of rows and columns\n");
    scanf("%d %d",&iValue1, &iValue2);

    Pattern(iValue1, iValue2);

    return 0;
}

File: Assignment___50/Program3.java
//_______________________________________________________________________________________________________________________________
//
//  File name :     Program3.java
//
//  Description :   Write a program which accept number of rows and number of columns from user and display below pattern.
//                  
//                  Input :     iRow = 6      iCol = 6
//                  
//                  Output :     
//                              *   *   *   *   *   *                                      
//                              *               *   *                                      
//                              *           *       *                                      
//                              *       *           *                                      
//                              *   *               *                                      
//                              *   *   *   *   *   *                                      
//
//  Author :        Sarika Devidas Gadhe
//
//_______________________________________________________________________________________________________________________________


import java.util.*;

class Pattern
{
    public void Pattern(int iRow , int iCol)
    {
        int i = 0, j = 0;

        for(i = iRow; i >= 1; i--)
        {
            for(j = 1; j <= iCol; j++)
            {
                if(i == j || i == 1 || i == iRow || j == 1 || j == iCol )
                {
                    System.out.print("*"+"\t");
                }
                else
                {
                    System.out.print("\t");
                }
                
            }
            System.out.println();
        }
       
    }
}

class Program3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue1 = 0, iValue2 = 0;

        System.out.println("Enter number of rows : ");
        iValue1 = sobj.nextInt();
        
        System.out.println("Enter number of columns : ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Pattern(iValue1,iValue2);

        sobj.close();


    }
}